translations/messages.pl.yml

```diff
@@ -474,7 +474,7 @@ quickstart:
         description: Jeżeli potrzebujesz pomocy, jesteśmy tutaj dla ciebie.
         github: na GitHubie
         email: przez e-mail
-        gitter: na Gitterze
+        matrix: na Gitterze
```

Comment:
I'm not sure we should keep the translation in the languages were there is a doubt on the update to make. I would go with letting a missing translation entry in this situation

Reply:
you'd prefer to remove these lines?

Reply:
Yes I would prefer

Reply:
Done.